mitmproxy/addons/proxyserver.py

```diff
@@ -144,6 +144,12 @@ def register_connection(
             del self.connections[connection_id]
 
     def load(self, loader):
+        loader.add_option(
+            "store_streamed_bodies",
+            bool,
+            False,
+            "Store HTTP request and response bodies when streamed.",
```

Comment:
Let's expand the docstring a bit. How about this?

```suggestion
            "Store HTTP request and response bodies when streamed (see `stream_large_bodies`). "
            "This increases memory consumption, but makes it possible to inspect streamed bodies.",
```

We should also point to it in the `stream_large_bodies` docstring (which also should mention _request_ bodies while you're at it 🙏 ).

Reply:
Done in the latest commit